.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+# Context
+
+<!-- Is there a Trello ticket you can link to? -->
+<!-- Do you need to add any environment variables? -->
+<!-- Is an ADR required? An ADR should be added if this PR introduces a change to the architecture. -->
+
+# Changes in this PR
+
+## Screenshots of UI changes
+
+### Before
+
+### After
+
+# Release checklist
+
+As part of our continuous deployment strategy we must ensure that this work is
+ready to be released at any point. Before merging to `main` we must first
+confirm:
+
+## Pre merge checklist
+
+- [ ] Are any changes required to the e2e tests?
+- [ ] If you've added a new route, have you added a new
+  `auditEvent`? (see `server/routes/apply.ts` for examples)
+- [ ] Have all changes to any dependencies been deployed in a backwards
+    compatible way? (This includes our API, infrastructure, third-party
+    integrations and any secrets or environment variables)
```

Comment:
this feels a bit broad -- approaching "please confirm that you haven't forgotten anything"!

Perhaps it includes more direct questions which we could break out as separate items? e.g.

- are there environment variables or other infrastructure configuration which needs to be included in this release?
- are there any data migrations required. Automatic or manual?
- does this rely on changes being deployed to the CAS API?